.github/workflows/merge_to_main.yml

```diff
@@ -121,15 +121,19 @@ jobs:
           node-version: 22.12.0
       - run: npm install
       - run: cp .env.example .env
-      - name: Run tests and upload results to Trunk.io
+      - name: Run vitest
+        run: npm run test
+        continue-on-error: true
```

Comment:
suggestion: Could we leave a note that this is coupled with the `if: ${{ !cancelled() }}` step below as per your comment?